packages/core/src/redux/slices/rtcConnection/index.ts

```diff
@@ -11,7 +11,8 @@ import RtcManagerDispatcher, {
 import { createReactor, startAppListening } from "../../listenerMiddleware";
 import { selectRemoteParticipants, streamStatusUpdated } from "../remoteParticipants";
 import { StreamState } from "../../../RoomParticipant";
-import { selectAppWantsToJoin } from "../app";
+import { selectAppIsNodeSdk, selectAppWantsToJoin } from "../app";
+import { Chrome111 as MediasoupDeviceHandler } from "mediasoup-client/lib/handlers/Chrome111";
```

Comment:
```suggestion
import { Chrome111 as MediasoupDeviceHandler } from "mediasoup-client/lib/handlers/Chrome111.js";
```

You need to specify the .js extension to make it work with es modules. (The type: module pkg json change is in [another PR](https://github.com/whereby/sdk/pull/108) though, we can add it there too 🤷 )